urdf/ur.ros2_control.xacro

```diff
@@ -31,12 +31,12 @@
         <xacro:if value="${sim_ignition}">
           <plugin>ign_ros2_control/IgnitionSystem</plugin>
         </xacro:if>
-        <xacro:if value="${use_mock_hardware}">
+        <xacro:if value="${use_fake_hardware}">
           <plugin>mock_components/GenericSystem</plugin>
-          <param name="mock_sensor_commands">${mock_sensor_commands}</param>
+          <param name="fake_sensor_commands">${fake_sensor_commands}</param>
```

Comment:
I actually think this line is where `ros2_control` actually did introduce an incompatibility.

You can keep your own xacro argument named as `fake_sensor_commands`, but you may need to do something like:

```suggestion
          <param name="mock_sensor_commands">${fake_sensor_commands}</param>
```

... but please test it!

Reply:
Ah, actually I see now that they do still check `fake_sensor_commands`... so my segfault must be something else. Bummer. Sorry about that.